<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Injury Update</string>
  </property>
  <widget class="QTableWidget" name="tblPlayers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>511</width>
     <height>671</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Club</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Player</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Club</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Player</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Injury Length</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLeague">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblReturned">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Returned from injury</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNew">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New injuries</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
